vcell-admin/resources/log4j2.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN" name="XMLConfigTest" packages="org.vcell.*">
+    <Properties>
```

Comment:
@CodeByDrescher take a look at this - FYI this will show up in the class loader as a default (not sure if JSBML or vcell-admin version will be picked up as the default.

Maybe we should put a default into vcell-core instead.

I'll accept this today - but we should rethink it.

Reply:
@jcschaff @CodeByDrescher sorry, that was for my own running config, I accidentally added to Git staging

Reply:
I'll remove before merge